packages/itwinui-react/src/core/utils/components/Divider.tsx

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../hooks';
+import '@itwin/itwinui-css/css/utils.css';
+
+export type DividerProps = React.ComponentPropsWithRef<'hr'>;
+
+/**
+ * Shows a divider
+ */
+export const Divider = React.forwardRef<HTMLHRElement, DividerProps>(
+  (props, ref) => {
+    const { className, ...rest } = props;
```

Comment:
Where is 'aria-orientation' prop?

Reply:
This prop has been added and tests have been written. However, vertical dividers are not shown @FlyersPh9 

![image](https://user-images.githubusercontent.com/78564221/224441629-f43bbfd0-74e5-4fcd-8311-f0e1e9601c6e.png)

Reply:
Sorry @adamhock, just seeing this now.  Where can I reproduce this demo in Storybook?

Reply:
Fixed in #1044 with https://github.com/iTwin/iTwinUI/pull/1044/commits/ccc24835a5f3bfaf1377b7f94c7479f79a35f418